front/src/reducers/osrdconf/stdcmConf/index.ts

```diff
@@ -25,6 +25,18 @@ export const stdcmConfSlice = createSlice({
       state.originTime = stdcmConfInitialState.originTime;
       state.speedLimitByTag = stdcmConfInitialState.speedLimitByTag;
     },
+    updateTotalMass(
+      state: Draft<OsrdStdcmConfState>,
+      action: PayloadAction<OsrdStdcmConfState['totalMass'] | null>
+    ) {
+      state.totalMass = action.payload === null ? undefined : action.payload;
+    },
+    updateTotalLength(
+      state: Draft<OsrdStdcmConfState>,
+      action: PayloadAction<OsrdStdcmConfState['totalLength'] | null>
+    ) {
+      state.totalLength = action.payload === null ? undefined : action.payload;
+    },
```

Comment:
I think we don't need to specify `null` in the type. Simplifies the code
```suggestion
    updateTotalMass(
      state: Draft<OsrdStdcmConfState>,
      action: PayloadAction<OsrdStdcmConfState['totalMass']>
    ) {
      state.totalMass = action.payload;
    },
    updateTotalLength(
      state: Draft<OsrdStdcmConfState>,
      action: PayloadAction<OsrdStdcmConfState['totalLength']>
    ) {
      state.totalLength = action.payload;
    },
```